lib/utils.js

```diff
@@ -161,11 +161,20 @@ module.exports.mapToMac = function (options) {
 module.exports.actionJackerDecorator = function (emitter, options, fn, mapper) {
   options = clone(options);
   fn = fn || function (err, data) {};
-  return function (err, data) {
-    fn.apply(emitter, [err, data]);
-    if (err || !mapper || !data) return;
-
-    var key = mapper(data);
+  return function (err, data) {
+
+    var resultantData = data;

```

Comment:
Sure, it can be `undefined`, but I was thinking of why storing it in a new variable?
This is effectively the same as:

```js
    // Sanitize the data
    if(data) {
      data = data.toLowerCase().trim();
      if(data.match(/^activate/)) {
        data = 'activate';
      }
    }
    fn.apply(emitter, [err, data]);
    if (err || !mapper || !data) return;

    var key = mapper(data);
    if (!key) return;
    emitter.emit(key, emitter, options);
```